### DISCUSSION TITLE
Add confirm before bulk action

### DISCUSSION THREAD

**Discussion by Jazzplunker** [rappasoft/laravel-livewire-tables#414] [Feature Requests]
How about add confirm before do bulk action,
This can help if i have bulk action for delete

I have some sample in my mind if you interested:
```
public array $bulkActions = [
    'delete' => 'label:Delete|confirm:are you sure?',
];
```

**Comment by vmosoti** [comment]
This you can implement yourself using sweetalerts or modals

**Comment by lrljoe** [comment]
Is in V3
